code/_helpers/global_lists.dm

```diff
@@ -118,3 +119,27 @@ var/list/playable_species = list() // A list of ALL playable species, whiteliste
 /proc/get_playable_species()
 	build_species_lists()
 	. = global.playable_species
+
+/proc/get_valid_teleporter_beacons()
+	var/list/atom/result = list()
```

Comment:
Typing the list var doesn't actually do anything, just for the record.
You could also avoid an alloc by using `.`.

Reply:
ex.
````
/proc/get_valid_teleporter_beacons()
	for(var/B in teleporter_beacons)
		...
		LAZYADD(., B)
````